Packs/Jira/Integrations/JiraV2/JiraV2.py

```diff
@@ -317,18 +317,27 @@ def get_account_id_from_attribute(
         override user identifier for Jira v2 API
         https://docs.atlassian.com/software/jira/docs/api/REST/8.13.15/#user-findUsers
         """
-        users = search_user(attribute, max_results, is_jirav2api=True)
+        users = list(search_user(attribute, max_results, is_jirav2api=True))
         account_ids = {
             user.get('name') for user in users if (attribute.lower() in [user.get('displayName', '').lower(),
                                                                          user.get('emailAddress', '').lower()])}
     else:
-        users = search_user(attribute, max_results)
+        users = list(search_user(attribute, max_results))
         account_ids = {
             user.get('accountId') for user in users if (attribute.lower() in [user.get('displayName', '').lower(),
                                                                               user.get('emailAddress', '').lower()])}
 
     if not account_ids:
-        return f'No Account ID was found for attribute: {attribute}.'
+        if not users:
+            return f'No Account ID was found for attribute: {attribute}.'
+        if len(users) == 1:
+            account_ids = {users[0].get('name')} if is_jirav2api == 'true' else {users[0].get('accountId')}
+        else:
+            demisto.debug(f'Account IDs found:\n {account_ids}')
```

Comment:
```suggestion
            demisto.debug(f'f'Multiple account IDs found, but it was not possible to resolve which one of them is most relevant to attribute \"{attribute}\". Account ids: {account_ids}')
```
I removed the new line because we should not add new lines in logs. It makes it harder to search the logs and filter by lines.